composer.json

```diff
@@ -40,5 +40,10 @@
     "php-http/message": "<1.13.0"
   },
   "prefer-stable": true,
-  "minimum-stability": "dev"
+  "minimum-stability": "dev",
+  "config": {
+    "allow-plugins": {
+      "php-http/discovery": true
+    }
+  }
```

Comment:
What's this new plugin?

Reply:
It showed up when I ran `composer update -W`

It's this https://github.com/php-http/discovery/blob/1.x/src/Composer/Plugin.php

Looks like it finds something standard to make an HTTP request with if there isn't something already installed

Reply:
It's a requirement of knplabs/github-api which is itself a requirement in the root composer.json

I'm running PHP 8.1, rather than PHP 7.4, which is possibly why it only showed up now